demos/modals/modals.css

```diff
@@ -0,0 +1,5 @@
+body {
```

Comment:
for something like this, maybe a common demo.css file would be better, than separate stylesheets for everything?